src/test/datascience/debugger.vscode.test.ts

```diff
@@ -98,7 +98,7 @@ suite('Run By Line @debugger', function () {
     // Cleanup after suite is finished
     suiteTeardown(() => closeNotebooksAndCleanUpAfterTests(disposables));
 
-    test('Delete temp debugging files', async function () {
+    test('Delete temp debugging files @mandatory', async function () {
```

Comment:
If you're parsing this in CI or something, it might be nicer if you had a util to make it more strongly typed? `testMandatory(...)`

Reply:
cc @DonJayamanne

Reply:
https://github.com/microsoft/vscode-jupyter/issues/12373